content/doc/book/managing/plugins.adoc

```diff
@@ -135,12 +140,13 @@ restarted in order for the changes to take effect.
 ==== On the controller
 
 Assuming a `.hpi` file has been explicitly downloaded by a system
-administrator, the administrator can manually place the `.hpi` file in a
+administrator, the administrator can manually place the file in a
 specific location on the file system.
 
 Copy the downloaded `.hpi`` file into the `JENKINS_HOME/plugins` directory on
 the Jenkins controller (for example, on Debian systems `JENKINS_HOME` is generally
 `/var/lib/jenkins`).
+If an administrator manually copies a plugin archive into the `plugins` directory, it should be named with a `.jpi` suffix to match the file names used by plugins installed from the update center.
```

Comment:
Second occurrence of this. Deliberate to already mention this in the introduction?